### PR TITLE
feat(Component/Breadcrumb): improve a11y

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
1. no landmark
2. first items dropdown trigger has no explicit alternative text (only the ellipsis)
3. items links need a more explicit text than its content

**What is the chosen solution to this problem?**
1. adding a navigation role is not compatible with our theme. Still no landmark but wrap component with a `<nav>` providing navigation info
2. add an aria-label (i18n)
3. set the title (tooltip) content as aria-label

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js [hunk 1]
```diff
@@ -71,6 +74,7 @@ function Breadcrumbs(props) {
 						bsStyle="link"
 						role="link"
 						title={title || text}
+						aria-label={title}
```
Comment (frassinier) [inline packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: Why adding an aria-label if a label and a title are already provided? 
Which information is added for screen readers?

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js [hunk 2]
```diff
@@ -109,15 +122,20 @@ function Breadcrumbs(props) {
 	}
 
 	return (
-		<ol id={props.id} className={classNames('breadcrumb', theme['tc-breadcrumb'], 'tc-breadcrumb')}>
+		<ol
+			id={props.id}
+			className={classNames('breadcrumb', theme['tc-breadcrumb'], 'tc-breadcrumb')}
+			role="navigation"
```
Comment (frassinier) [inline packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: ⚠️ Be careful with [that](https://github.com/Talend/ui/blob/master/packages/theme/src/theme/variations/_tdp.scss#L12)

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js (hunk 1 above)
Comment (jsomsanith) [reply packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: I assume that 
* `text` is the label, so the content of the button
* `title` the tooltip

The problem is that the tooltip is more accurate, providing more info than the content.
But `title` is less important than `content` in a11y text order. So screen readers will take the content over the title, so the less accurate/precise text.

So when you have `title` and `content`, we add an `aria-label` with the `title` value to be read by SR over the content.

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js (hunk 2 above)
Comment (jsomsanith) [reply packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: Oh thank you, I'll remove it on Breadcrumb

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js (hunk 1 above)
Comment (frassinier) [reply packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: Ok got it thx

File: packages/components/src/Breadcrumbs/Breadcrumbs.component.js (hunk 2 above)
Comment (frassinier) [reply packages/components/src/Breadcrumbs/Breadcrumbs.component.js]: Here is a good [resource](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html)

**Pull request merged**